requirements/datatype_image_extras.txt

```diff
@@ -10,3 +10,4 @@ learn2learn
 structlog==21.1.0   # remove when baal resolved its dependency.
 baal
 fastface
+fairscale
```

Comment:
how is this required?

Reply:
VISSL tests require it but somehow it isn't listed in their requirements. I think because it used to get installed with PL maybe?